content/docs/14-troubleshooting/06-nodes.md

```diff
@@ -9,13 +9,13 @@ fullWidth: false
 
 # Nodes & Clusters
 
-This page is covers common debugging scenarios for nodes and clusters post deployment.
+This page covers common debugging scenarios for nodes and clusters post-deployment.
```

Comment:
```suggestion
This page covers common debugging scenarios for nodes and clusters after they have been deployed.
```

Reply:
Here, it's correct to exclude the hyphen because there's nothing after post deployment to modify, but the suggested change might be preferable. 

For example, a hyphen is correct in step 6 because 'read-only' modifies 'file system':

"Reboot the system to resume the default read-only file system."